### ISSUE TITLE
Showing completely black window on running flutter run in linux

### ISSUE THREAD

**Issue by rohansen856** [foss42/apidash#492]
#### Describe the bug/problem
When i clone the repo and run the flutter project in linux device, it shows a completely black unresponsive screen.
![Screenshot from 2024-11-06 18-34-26](https://github.com/user-attachments/assets/a05ed99c-be68-45ba-97aa-29bfb5749cd0)

#### Steps to Reproduce the bug/problem
- clone the repo locally
- run the command: `flutter run` (for linux)

#### Expected behavior
- the app should start normally

#### Device Info (The device where you encountered this issue)
 - OS: Ubuntu
 - Version: 22.04LTS
 - Browser: None

#### Flutter Doctor
Please run the `flutter doctor -v` command and provide the details below:

```
[✓] Flutter (Channel stable, 3.24.4, on Ubuntu 24.04.1 LTS 6.8.0-48-generic, locale en_US.UTF-8)
    • Flutter version 3.24.4 on channel stable at /home/rcsen/snap/flutter/common/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 603104015d (13 days ago), 2024-10-24 08:01:25 -0700
    • Engine revision db49896cf2
    • Dart version 3.5.4
    • DevTools version 2.37.3

[✓] Android toolchain - develop for Android devices (Android SDK version 35.0.0)
    • Android SDK at /home/rcsen/Android/Sdk
    • Platform android-35, build-tools 35.0.0
    • Java binary at: /snap/android-studio/161/jbr/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.10+0-17.0.10b1087.21-11609105)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at google-chrome

[✓] Linux toolchain - develop for Linux desktop
    • clang version 10.0.0-4ubuntu1
    • cmake version 3.16.3
    • ninja version 1.10.0
    • pkg-config version 0.29.1

[✓] Android Studio (version 2024.1)
    • Android Studio at /snap/android-studio/161
    • Flutter plugin version 82.0.2
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.10+0-17.0.10b1087.21-11609105)

[✓] VS Code (version 1.95.0)
    • VS Code at /usr/share/code
    • Flutter extension version 3.100.0

[✓] VS Code (version 1.96.0-insider)
    • VS Code at /usr/share/code-insiders
    • Flutter extension can be installed from:
      🔨 https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[✓] Connected device (2 available)
    • Linux (desktop) • linux  • linux-x64      • Ubuntu 24.04.1 LTS 6.8.0-48-generic
    • Chrome (web)    • chrome • web-javascript • Google Chrome 130.0.6723.91

[✓] Network resources
    • All expected network resources are available.

• No issues found!
```

**Comment by ashitaprasad** [comment]
Thanks for reporting the issue. Will post an update after recreating the issue.

**Comment by ashitaprasad** [comment]
We have successfully recreated the issue.

**Comment by ashitaprasad** [comment]
This is the issue with the Flutter framework and it was reported in https://github.com/flutter/flutter/issues/151098
A fix has been made and it will be available in flutter 3.27.0 https://github.com/flutter/engine/commit/a02799141d7501318f205a0eb1cffe5c7e1c7193

Currently, a workaround is to launch the Linux app in **Profile** mode instead of Debug mode.
